pallets/messages/src/benchmarking.rs

```diff
@@ -9,6 +9,7 @@ use frame_system::RawOrigin;
 const MESSAGES: u32 = 499;
 const SCHEMAS: u32 = 50;
 const IPFS_SCHEMA_ID: u16 = 65535;
+const IPFS_PAYLOAD_LENGTH: u32 = 10;
```

Comment:
I'm not sure how accurately this represents (or should represent) an offchain payload length, but would we want to revisit the benchmarks later?